README.md

```diff
@@ -1,24 +1,73 @@
+Authors, AD, WG chairs, and document shepherd, please ensure that your GitHub notification settings are set such that you can participate promptly in discussions of issues and pull requests. For more information, please see [Configuring notifications](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications).
 
+This document is being edited in Markdown.  Once the content is stable, the Markdown will be converted to XML and formatted as an RFC.  Your final approval means you approve both the content and format.
 
-## Contributing
+## Issue Tracking
+The initial version of rfc9345-to-be.md is a copy of draft-ietf-tls-subcerts.md that was approved for publication. 
+
+The current version of rfc9345-to-be.md has been edited minimally for correctness. See PR #NNN.
+
+The initial version of the file has been edited minimally for correctness.  The RFC Production Center's suggested updates have been added as PRs.  Open-ended questions and FYIs have been added as issues.  Please accept the PRs or propose alternate updates.  For the issues, provide your answers in the comments or add PRs to address the issues.  Use the issue labels to indicate their status:
+* rfced - initial label of RFC Editor questions and FYIs (keep this label if the issue is still active).
+* question - a follow-up question has been added to the issue discussion. Use @mention to identify the participant who should answer. 
+* AD approval required - the resolution of the issue requires the AD to review and approve the change.
+* rfced-ready - the RFC Editor can update the document and/or close the issue based on the issue discussion.
+
+Feel free to add new issues and PRs for new edits and questions. 
+
+The editor will close the issue when it has been addressed either by accepting a pull request or simply closing the issue if no changes are needed. 
+
+## Pull Requests
+If you would like to submit changes, please create a pull request and identify one or more issues that the pull request closes in the comments. 
+
+If the editor creates the PR, the editor will expect feedback from the reviewer before merging the PR. 
+
+
+## Viewing Markdown Outputs
+Output files may be viewed here:
+* [https://www.rfc-editor.org/authors/rfc9345.txt](https://www.rfc-editor.org/authors/rfc9345.txt)
+
+Once the content of the .md file is stable and an XML file has been created, the following files will also be provided:
+* [https://www.rfc-editor.org/authors/rfc9345.html](https://www.rfc-editor.org/authors/rfc9345.html)
+* [https://www.rfc-editor.org/authors/rfc9345.pdf](https://www.rfc-editor.org/authors/rfc9345.pdf)
+* [https://www.rfc-editor.org/authors/rfc9345.xml](https://www.rfc-editor.org/authors/rfc9345.xml)
+   
+To create your own output files, use the [tools webservice](https://author-tools.ietf.org/) and select one of the output options.
+
+## Viewing Diffs
```

Comment:
If we're using this repo as opposed to an RPC one, this section seems unnecessary.  We'll just use PR diffs, as in this one.